WcaOnRails/app/models/competition.rb

```diff
@@ -15,18 +15,23 @@ class Competition < ActiveRecord::Base
   has_many :organizers, through: :competition_organizers
   has_many :media, class_name: "CompetitionMedium", foreign_key: "competitionId", dependent: :delete_all
 
+  CLONEABLE_ATTRIBUTES = %w(cityName countryId information eventSpecs venue venueAddress venueDetails website latitude longitude contact remarks use_wca_registration guests_enabled).freeze
+  UNCLONEABLE_ATTRIBUTES = %w(id name year month day endMonth endDay cellName showAtAll isConfirmed registration_open registration_close results_posted_at).freeze
```

Comment:
This looks really awesome now, Jeff!

Could you split these 2 lines into one line per attribute? That means that future diffs will be easier to read, and the chances of conflicts are smaller. As we work on #329, I suspect that Competitions will continue to gather more and more attributes.